drakrun/drakrun/apiscout.py

```diff
@@ -0,0 +1,116 @@
+import json
+import logging
+from operator import attrgetter
+from pathlib import Path, PureWindowsPath
+import pefile
+from typing import List, Dict, Any
+
+log = logging.getLogger(__name__)
+
+
+def get_bitness(pe: pefile.PE) -> int:
+    if pe.FILE_HEADER.Machine == pefile.MACHINE_TYPE["IMAGE_FILE_MACHINE_AMD64"]:
+        return 64
+    elif pe.FILE_HEADER.Machine == pefile.MACHINE_TYPE["IMAGE_FILE_MACHINE_I386"]:
+        return 32
+    else:
+        log.error(
+            f"Unsupported machine_type: {pe.FILE_HEADER.Machine} -> {pefile.MACHINE_TYPE[pe.FILE_HEADER.Machine]}"
+        )
+        return 0
+
+
+def get_product_version(pe: pefile.PE) -> str:
+    """
+    Based on https://stackoverflow.com/a/16076661/12452744
+    """
+
+    def LOWORD(dword):
+        return dword & 0x0000FFFF
+
+    def HIWORD(dword):
+        return dword >> 16
+
+    if len(pe.VS_FIXEDFILEINFO) != 1:
```

Comment:
This can crash with:
```
Traceback (most recent call last):
  File "/opt/venvs/drakrun/lib/python3.7/site-packages/drakrun/draksetup.py", line 581, in create_rekall_profile
    local_dll_path
  File "/opt/venvs/drakrun/lib/python3.7/site-packages/drakrun/apiscout.py", line 60, in make_static_apiscout_profile_for_dll
    dll_entry["version"] = get_product_version(pe)
  File "/opt/venvs/drakrun/lib/python3.7/site-packages/drakrun/apiscout.py", line 34, in get_product_version
    if len(pe.VS_FIXEDFILEINFO) != 1:
AttributeError: 'PE' object has no attribute 'VS_FIXEDFILEINFO'
```